[BUG][Test] Controller 추가 시 GlobalExceptionHandler 테스트 깨짐

## 어떤 버그인가요?
- Controller 추가 시 GlobalExceptionHandler 테스트 깨짐
- AiController 추가 브랜치에서 발견해 예시에 ai 패키지 존재
```
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'aiController' defined in file [/paths/be-nice-backend/build/classes/java/main/today/sesac/shoutify/ai/controller/AiController.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'today.sesac.shoutify.ai.service.AiService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
...
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'today.sesac.shoutify.ai.service.AiService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
```
<img width="515" alt="Image" src="https://github.com/user-attachments/assets/e3716424-d619-4b78-b788-c704f03f0b59" />


## 버그를 재현하기 위한 방법
- GlobalExceptionHandlerTest 실행

## 정상 동작
- test 통과

## 버그 환경 
X

## 참고
- WebMvcTest는 모든 Controller 등록하기 때문에 발생한 에러로 추정, 해당하는 controller만 포함시킬 예정

--- a/src/test/java/today/sesac/shoutify/global/advice/GlobalExceptionHandlerTest.java
+++ b/src/test/java/today/sesac/shoutify/global/advice/GlobalExceptionHandlerTest.java
@@ -9,7 +9,6 @@ import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Import;
 import org.springframework.http.MediaType;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
@@ -28,8 +27,7 @@ import today.sesac.shoutify.global.exception.PermissionRequiredException;
 /**
  * 전역 예외 처리 테스트
  */
-@WebMvcTest
-@Import(GlobalExceptionHandler.class)
+@WebMvcTest(controllers = GlobalExceptionHandlerTest.class)
 class GlobalExceptionHandlerTest {
 
 	static final String IS_SUCCESS = "$.isSuccess";

